### DISCUSSION TITLE
What does "basana" mean ?

### DISCUSSION THREAD

**Discussion by xzmeng** [gbeced/basana#16] [General]
I have googled it, the first result seems to be a town in India, the following three results link to this project, the next one says
>Basana in India is the name of a plant defined with [Bothriochloa pertusa](https://www.wisdomlib.org/definition/bothriochloa-pertusa#biology) in various botanical sources.

So is it the right meaning of "basana" ?

**Comment by gbeced** [comment]
No, its just a word that my daughters used to say when they were babies and starting to speak.
